how-to/use-theming/client/src/provider.ts

```diff
@@ -9,6 +9,12 @@ window.addEventListener("DOMContentLoaded", async () => {
 	const platform = fin.Platform.getCurrentSync();
 	await platform.once("platform-api-ready", async () => bootstrap());
 
+	await fin.InterApplicationBus.subscribe({ uuid: "*" }, "theme-restart", async (payload) => {
```

Comment:
I think this should be locked down to fin,me.identity.uuid no?

Reply:
Couldn't get it to work with specific identity